mcs/class/System/Test/System.Net.NetworkInformation/IPInterfacePropertiesTest.cs

```diff
@@ -0,0 +1,73 @@
+//
+// IPInterfacePropertiesTest.cs - NUnit Test Cases for System.Net.NetworkInformation.IPInterfaceProperties
+//
+// Authors:
+//   Ben Woods (woodsb02@gmail.com)
+//
+
+using NUnit.Framework;
+using System;
+using System.Net;
+using System.Net.NetworkInformation;
+
+namespace MonoTests.System.Net.NetworkInformation
+{
+
+[TestFixture]
+public class IPInterfacePropertiesTest
```

Comment:
I followed the template and examples listed here (neither of which indent this): http://www.mono-project.com/community/contributing/test-suite/
https://github.com/mono/mono/blob/master/mcs/class/doc/TemplateTest.cs
https://github.com/mono/mono/blob/master/mcs/class/corlib/Test/System/StringTest.cs